tests/errors/cpp_preincrement.pyx

```diff
@@ -0,0 +1,26 @@
+# mode: error
+
+cimport cython
+
+cdef extern from *:
+    cdef cppclass Foo:
```

Comment:
Could cpp_preincrement and cpp_postincrement be combined into one test? (I think it should be possible if the errors appear at the same stage). Largely because (I think) one bigger test file is quicker to run than two smaller test files.

If not possible it's fine though.

Reply:
I merged them